spring-integration-graphql/src/main/java/org/springframework/integration/graphql/outbound/GraphQlMessageHandler.java

```diff
@@ -54,7 +57,7 @@
 	@Nullable
 	private Locale locale;
 
-	private String executionId = null;
+	private Expression idExpression = new SupplierExpression<>(() -> null);
```

Comment:
Let's make it more clear, so the name should be an `executionIdExpression`.
And this one can be as default expression: `new FunctionExpression<Message<?>>(message -> message.getHeaders().getId())`